### DISCUSSION TITLE
can I create double csi driver for rook,one connect local,another connect remote rook cluster

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13726] [Q&A] [answered]
can I create double csi driver for rook,one connect local,another connect remote rook cluster

**Comment by Madhu-1** [comment, marked as answer]
You dont need 2 csi drivers for 2 ceph clusters.  if you deploy rook and import/create 2 ceph clusters a single csi driver can serve 2 ceph clusters/
